### PR TITLE
Templates base

### PR DESCRIPTION
**Pull request by mahmoudHussein**
@maizaied @HebaElSobky @mohamedahmedawad93 please check if the bootstrap is working and merge it.

### CODE REVIEW

File: project/tager/tager/settings.py [hunk 1]
```diff
@@ -115,6 +118,10 @@
     # Put strings here, like "/home/html/django_templates" or "C:/www/django/templates".
     # Always use forward slashes, even on Windows.
     # Don't forget to use absolute paths, not relative paths.
+
+    relative_project_path('templates'),
```
Comment (mohamedahmedawad93) [inline project/tager/tager/settings.py:121]: @mahmoudHussein you have a syntax error here

**Comment by mahmoudHussein** [comment]
@mohamedahmedawad93 @maizaied @HebaElSobky please review and merge the code !

**Comment by mohamedahmedawad93** [comment]
@mahmoudHussein your code is valid :)

**Comment by HebaElSobky** [comment]
@mahmoudHussein scenario is valid for registered users #75

**Pull request merged**